stack/create.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2022 Mineiros GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stack
```

Comment:
feel free to disregard but for me it's strange to place this stack creation API here, as it's very UI/cli related...

Reply:
hmm I disagree, it is a function to create a stack on the filesystem, for me this would make code generation also something very UI/CLI related...it is just generating code as other features do, and in this case it is generating a stack, I don't see the mismatch :thinking:

Reply:
For example, maybe there is some way to provide a cool "create a stack" integrated on vscode, then to do that you would need to import a package inside cli...which would be very odd IMO, because the feature itself is not coupled to how the user will interact with it (could be a CLI, some IDE integration, etc).

Reply:
> for me this would make code generation also something very UI/CLI related...

the code generation works based on the whole project configuration, the input is the configuration, it's a core feature of terramate. The "create config" here will have to change whenever we change the semantics of the cli arguments of the `terramate create` command...